charts/kpow/templates/deployment.yaml

```diff
@@ -37,7 +37,7 @@ spec:
           env:
             {{- range $key, $val := . }}
             - name: {{ $key }}
-              value: {{ $val }}
+              value: {{ $val | quote }}
```

Comment:
Discovered this when attempting to set `SAMPLER_CONSUMER_THREADS` value.  Helm throws an error setting any environment variable to an integer value.  This solved the problem.